app/uk/gov/hmrc/mdgstub/controllers/MdgStubController.scala

```diff
@@ -19,47 +19,51 @@ package uk.gov.hmrc.mdgstub.controllers
 import java.io.StringReader
 import java.time.Instant
 
+import akka.actor.ActorSystem
 import javax.inject.{Inject, Singleton}
 import org.xml.sax.SAXParseException
 import play.api.Logger
-import play.api.libs.concurrent.Promise
 import play.api.mvc._
-import uk.gov.hmrc.mdgstub.util.{Eithers, PerfLogger}
-import uk.gov.hmrc.play.bootstrap.controller.BaseController
+import uk.gov.hmrc.mdgstub.util.Eithers
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
 import scala.util.{Failure, Success, Try}
 import scala.xml._
 
 @Singleton()
-class MdgStubController @Inject() (implicit ec: ExecutionContext) extends BaseController {
+class MdgStubController @Inject() (actorSystem: ActorSystem, cc: ControllerComponents)
+                                  (implicit ec: ExecutionContext) extends BackendController(cc) {
 
+  private val logger = Logger(this.getClass)
   private val availabilityMode = raw"(\d{3}):(\d+):(\d+)".r
 
+  // Use akka.pattern.after(duration, actorSystem.scheduler)(Future(message)) instead
```

Comment:
Is this comment still needed now that you've implemented this pattern in the function below?

Reply:
Ta.  Tripped up by my own prompt ...